src/components/datatable/DocumentsTable.vue

```diff
@@ -95,12 +98,52 @@
     methods: {
       onGridReady(params) {
         this.gridApi = params.api;
+        let sort_model = [];
+        const sort_URL = this.$route.query['sort'];
+        if (sort_URL) {
+          let sort_item = {};
```

Comment:
there is no check on validity of the URL. Could a regexp be used, e.g. ?

Reply:
I had a look at this, the URL is already slightly error-tolerant, the code tries to parse the URL and rewrite the sort model. If a column name is not spelled correctly, the sort instruction is dropped and the URL updated.
What would you expect to do after validity check ? User-feedback ? Isn't it better to have a silent robust mechanism. Or are you afraid of a vulnerability ? The query is parsed twice afterwards (split tokens, configure sort model) with no active string, so I don't see a way to produce a dangerous behaviour, but I am very bad in security topics, so I may overlook something.